### PR TITLE
Us 111

### PR DESCRIPTION
**Pull request by edduardaq**

### CODE REVIEW

File: db/mongo_client.py
Comment (ebarahonautn) [inline db/mongo_client.py:1]: @DevWilliamCN delete the  **update_verification_code()** method, you can use the method **update_by_condition()**

File: controllers/auth/VerifyCodeController.py
Comment (ebarahonautn) [inline controllers/auth/VerifyCodeController.py:1]: @DevWilliamCN this is a **PUT** and not a POST

- This controller needs to be created in **controllers/user/UserVerificationController.py**
-  the correct route is **/user/verification**
- Delete this code:    ` STATIC_VERIFICATION_CODE = 123456`  # Código de verificación estático para pruebas
- the method no apply all changes in the DB, please review carefully the user story: [https://ever-utn.atlassian.net/browse/US-111](https://ever-utn.atlassian.net/browse/US-111)

File: controllers/user/UserEnrollment_controller.py
Comment (ebarahonautn) [inline controllers/user/UserEnrollment_controller.py:1]: @DevWilliamCN you don't need to apply changes in this controller

File: models/auth/auth.py
Comment (ebarahonautn) [inline models/auth/auth.py:1]: @DevWilliamCN the models/auth folder is not longer used and exists. Please revert this change.

File: models/user/user.py
Comment (ebarahonautn) [inline models/user/user.py:1]: @DevWilliamCN no use `__dbmanager__.update_data()`, better use `__dbmanager__.update_by_condition()`

change the method name of **save_to_db()** for this: **user_activation()**

**Review by ebarahonautn — CHANGES_REQUESTED** [review]

**Review by ebarahonautn — CHANGES_REQUESTED** [review]
@DevWilliamCN fix the conflicts you have in the PR

File: db/mongo_client.py
Comment (DevWilliamCN) [reply db/mongo_client.py:1]: Listo profe, fue corregido

File: controllers/auth/VerifyCodeController.py
Comment (DevWilliamCN) [reply controllers/auth/VerifyCodeController.py:1]: Ready teacher, it was corrected

File: controllers/user/UserEnrollment_controller.py
Comment (DevWilliamCN) [reply controllers/user/UserEnrollment_controller.py:1]: Ready teacher, it was corrected

File: models/auth/auth.py
Comment (DevWilliamCN) [reply models/auth/auth.py:1]: Ready teacher, it was corrected

File: models/user/user.py
Comment (DevWilliamCN) [reply models/user/user.py:1]: Ready teacher, it was corrected

File: service.py
Comment (ebarahonautn) [inline service.py:1]: @DevWilliamCN you don't need to add the **UserVerificationController**

File: controllers/auth/VerifyCodeController.py
Comment (ebarahonautn) [reply controllers/auth/VerifyCodeController.py:1]: @DevWilliamCN this controller is not required. Please delete this file: **controllers/auth/VerifyCodeController.py**

**Review by ebarahonautn — CHANGES_REQUESTED** [review]

File: service.py
Comment (DevWilliamCN) [reply service.py:1]: Ready

Comment (ebarahonautn) [inline service.py:1]: @DevWilliamCN I was wrong, please add the following code in the service

**from controllers.user.user import UserVerificationController
api.add_resource(UserVerificationController, UserVerificationController.route)**

Sorry for my mistake.

**Review by ebarahonautn — CHANGES_REQUESTED** [review]

File: service.py
Comment (DevWilliamCN) [reply service.py:1]: It's ready

Modification completed

**Pull request merged**